app/views/videos/show.html.haml

```diff
@@ -12,3 +12,35 @@
           %a.btn.btn-primary(href="") Watch Now
           %a.btn.btn-default(href="") + My Queue
 
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            %label Rate this video
+            .row
+              .col-sm-3
+                =f.select :rating, options_for_select([5,4,3,2,1].map{|number| [pluralize(number, "Star")]}), class:"form-control"
```

Comment:
Its better when you turn `options_for_select` to a helper method. too much logic in this template.